applications/search/src/pages/concept-details-page/concept-details-page.jsx

```diff
@@ -311,6 +314,15 @@ export const ConceptDetailsPage = ({
     description: getTranslateText(_.get(conceptItem, ['definition', 'text']))
   };
 
+  const source = {
+    sourceRelationship: _.get(
```

Comment:
import get from 'lodash/get'

Reply:
Requires installation of a new dependency. Will not fix in this branch.